src/AnnotationThread.js

```diff
@@ -165,29 +155,32 @@ class AnnotationThread extends EventEmitter {
             pageEl.appendChild(popoverLayer);
         }
 
-        const isPending = !firstAnnotation;
+        const isPending = !this.threadNumber;
         if (isPending) {
             this.state = STATES.pending;
         } else {
             this.state = STATES.active;
         }
 
+        this.save = this.save.bind(this);
+        this.updateTemporaryAnnotation = this.updateTemporaryAnnotation.bind(this);
+        this.delete = this.delete.bind(this);
```

Comment:
Do these need to be bound here, or can they be defined with arrow functions, as we do in other React codebases?

```
class MyThing {
  save = () => {
    // do stuff
   }
}

Reply:
See comment below